utils.sh

```diff
@@ -164,16 +164,16 @@ resource "ironic_node_v1" "openshift-master-${master_idx}" {
     }
   ]
 
-  properties {
-    "local_gb" = "${local_gb}"
-    "cpu_arch" =  "${cpu_arch}"
```

Comment:
Should these have been removed?

Reply:
Oops no, that's a rebase/debugging mistake, will fix